tools/palletjack2unbound

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env ruby
+
+# Write DNS resolver stub declarations from a Palletjack warehouse
+
+require 'palletjack'
+require 'optparse'
+require 'dns/zone'
+require 'ip'
+
+$options = {}
+
+def exists_ok(&code)
+  code.call()
+rescue Errno::EEXIST
+  nil
+end
+
+def stub_zone(zone, *stub_addrs)
+  head = "################################################################
+# Automatically generated by palletjack2unbound from 
+# #{$options[:warehouse]}
+################################################################
+
+stub-zone:
+  name: #{zone}\n"
+
+  stub_addrs.inject(head) do |result, addr|
+    result << "  stub-addr: #{addr}\n"
+  end
+end
+
+opts = OptionParser.new
+opts.banner = "Usage: #{$PROGRAM_NAME} -w <warehouse> -d <output directory>
+
+Write DNS resolver stub declarations from a Palletjack warehouse
+into Salt state configuration for unbound.
+
+E.g.
+  palletjack2unbound -w /etc/salt/respository/warehouse \\
+    -o /etc/salt/repository/state/unbound/files
+"
```

Comment:
You want en empty line at the end here, otherwise the example runs into the options list.